crates/worker/src/partition/state_machine/command_interpreter/mod.rs

```diff
@@ -111,6 +111,9 @@ pub trait StateReader {
 pub(crate) struct CommandInterpreter<Codec> {
     // initialized from persistent storage
     inbox_seq_number: MessageIndex,
+    /// Position of the start of outbox commands
+    outbox_head_seq_number: MessageIndex,
```

Comment:
Not super happy with this naming but I couldn't come up with anything better; I'd like to rename this and the tail to be explicit - will revisit before merging.